packages/app/package.json

```
@@ -69,7 +69,7 @@
       "targetSdk": 29,
       "compileSdk": 29,
       "buildTools": "29.0.3",
-      "firebase": "25.7.0",
+      "firebase": "25.12.0",
       "iid": "20.2.1",
```

```suggestion
      "iid": "20.3.0",
```

ios is at 6.34.0 also - that's worth a change I think so they are both up to current together

playServicesAuth is at 18.1.0 https://developers.google.com/android/guides/releases#july_16_2020

with those changes I think this will be good to go - have you checked for references to the versions anywhere else in the codebase? Just curious - I think I centralized them all here last time I did this but it was recent enough I'll scan again if you didn't look

Thanks!

---

I've updated the libraries you mentioned and the versions in docs too.
But I'm not sure if [this line](https://github.com/invertase/react-native-firebase/blob/fd1a2be6b6ab1dec89e5dce1fc237435c3e1d510/scripts/_TEMPLATE_/RNFB_Template_.podspec#L19)  needs to be updated or not

---

ah good find on the template - I mean, in the sense that you clearly scanned everywhere, thanks - I need to delete that directory though, the template is currently unsupported and scheduled for deletion - let me check through this again it might be good to go